template/default.tmpl

```diff
@@ -190,3 +190,6 @@ Alerts Resolved:
 {{ end }}
 {{ end }}
 {{ define "pushover.default.url" }}{{ template "__alertmanagerURL" . }}{{ end }}
+
+{{ define "jira.default.summary" }}{{ template "__subject" . }}{{ end }}
```

Comment:
Not sure how this ticket title will look like... How we could distinguish that from the manually created ticket?